eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgUtils.scala

```diff
@@ -119,6 +123,14 @@ object PgUtils extends JdbcUtils {
         }
       }
 
+      override def releaseExclusiveLock(implicit ds: DataSource): Unit = {
+        // put a new lease that expires right away: same as releasing the lock
+        obtainDatabaseLease(instanceId, 1 millisecond) match {
```

Comment:
Interesting, there is no cleaner mechanism to release a lock?

Reply:
In theory we could also delete the line in the lease table, but it's not better, because (1) we lose information (identifier of the last lock holder), (2) deleting from the lease table is a new scenario (currently we only insert or update) and (3) it's more code.